examples/07-todoMVC/components/TodoList.js

```diff
@@ -0,0 +1,26 @@
+import isolate from "@cycle/isolate"
+import TodoItem from "./TodoItem"
+import { liftListById, flatCombine, flatMerge, R, L } from "stanga"
+import { ul } from "@cycle/dom"
+
+export function TodoList ({ DOM, M }) {
+  const liftTodoItem = (id, todoItem$) => {
+    return isolate(TodoItem, `item-${id}`)({
+      DOM,
+      M: todoItem$,
+      parentM: M.lens("list")
+    })
+  }
+
+  let todoItems$ = M.flatMap(state =>
```

Comment:
`flatMap` flattens the observable of observables and produces values from every flattened observable. `flatMapLatest` disposes the previous flattened observable and produces values only from the **latest** flattened observable. Usually this is the wanted behavior because we are not interested in values that are derived from the old state anymore. 

`shareReplay` just replays the latest value if multiple observables subscribe to the stream - lacking `shareReplay` with `flatMapLatest` is usually one the common reasons why e.g. your clicks don't emit any events (because the observable has already emitted it's initial value (=observable to be flattened) during the first subscription).